docs/tutorials/openvino-mlflow.md

```diff
@@ -0,0 +1,611 @@
+    - the default values for the workflow input parameters specify training the model for 30 epochs. This will take approximately 40 minutes and yield an accuracy close to 0.8. You can modify these parameters to experiment with different values.
+
+    To configure the FuseML workflow:
+
+    ```bash
+    $ fuseml workflow create examples/workflows/mlflow-multi-e2e.yaml
+    Workflow "mlflow-multi-e2e" successfully created
+    ```
+
+
+3. Assign the `mlflow-multi-e2e` workflow to the `keras` codeset:
+
+    ```bash
+    $ fuseml workflow assign -c keras -p cifar10 -n mlflow-multi-e2e
+    Workflow "mlflow-multi-e2e" assigned to codeset "cifar10/keras"
+    ```
+
+4. Wait for the workflow run to finish running:
+
+    By assigning the workflow to the codeset, a workflow run will be created. You can check the status
+    of the workflow run by running the following command:
+
+    ```bash
+    fuseml workflow list-runs
+    +----------------------------+------------------+----------------+------------+------------------+
+    | NAME                       | WORKFLOW         | STARTED        | DURATION   | STATUS           |
+    +----------------------------+------------------+----------------+------------+------------------+
+    | fuseml-cifar10-keras-lrshv | mlflow-multi-e2e | 19 seconds ago | ---        | Running          |
+    +----------------------------+------------------+----------------+------------+------------------+
+    ```
+
+    You can also see a detailed view of the workflow run through the Tekton dashboard. The URL where
+    the dashboard can be accessed was printed during the FuseML installation and has the form:
+    `http://tekton.<FUSEML_DOMAIN>`, where `FUSEML_DOMAIN` is the common _.nip.io_ domain name used for
+    all services. In our case, this is `http://tekton.34.118.93.249.nip.io`.
+
+
+    <figure markdown> 
+      ![Workflow WIP In Tekton](img/openvino-mlflow/tekton-wip.png)
+      <figcaption>View of the Workflow in Progress in the Tekton Dashboard</figcaption>
+    </figure>
+
+    Aside from the long training time, note that since this is the first time the workflow is running, it will build a docker image including the software dependencies for training the model which may take a while. However, consecutive runs will skip that step as long as the dependencies are kept the same.
+
+    <figure markdown> 
+      ![Workflow Done In Tekton](img/openvino-mlflow/tekton-done.png)
+      <figcaption>View of the Completed Workflow in the Tekton Dashboard</figcaption>
+    </figure>
```

Comment:
Some comment about those images, it is really hard to see anything without zooming in.

Reply:
I added a featherlight extension to help with that